src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -722,7 +722,7 @@ public string AstIdentifierGuid
         {
             get
             {
-                return GUID.ToString().Replace("-", string.Empty).ToLower();
+                return GUID.ToString("N");
```

Comment:
what does the "N" do, does it also accomplish the toLower field? Why are you changing this AST id - we don't serialize this I don't think, it should only be used internally for the VM...